### ISSUE TITLE
Potential change to the judgment process and how to deal with inclarities in the rules

### ISSUE THREAD

**Issue by malte-l** [VeNomic-Administration/nomic#17]
tl;dr:
* Do you prefer the legal side (ie. interpreting rules) of the game or the game mechanics (ie. making rules that do new things, doing the new things that rules introduced)?
* If people tent to prefer the latter, perhaps it might be useful to change the judgment etc. process. Instead:
* There is a court, whose members don't change
* Those members are not players (so don't have a score, can't win, can't vote, etc.) and have to be neutral and uninvolved
* The court is tasked to judge over disagreements on the rules (in a game and fun enabling way) and perhaps also give advice on "legal matters" concerning the rules to help consistent gameplay.
* I would be interested in comments even if you did not read the rest.

That I write this now is a reaction to the situation https://github.com/VeNomic-Administration/nomic/pull/16.
It is possible suggestion/proposal, but I am mostly writing it down here so that the idea is out there, I don't necessarily want to push it / convince other people or so (though I will of course answer comments about it).

May I first pose a question to the other players: Do you enjoy the part of the game about trying to interpret the precise meaning of rules in cases such as this, arguing about it (and in the end perhaps having a judgment or people agreeing in the end) [called the first part of the game from now on], or would you prefer to focus more on moving your pieces on the board, creating rules that add items or other such game mechanisms, giving people hats of good karma, etc. [called the second part of the game from now on]?

In case people prefer the second part of the game, may I repeat a suggestion I made before this (second) game was started:
Instead of having this judgment system with three non-involved players judging etc. we could have one fixed "court" that rules on all cases, consisting of one or more fixed people that don't change (at least not on a turn-by-turn basis). The court is tasked with creating a consistent interpretation of the ruleset (in the spirit of playability etc.) when judging on disagreements, and is bound to a certain extent by precedents (thereby making judgment more predictable). The court is of course supposed to be neutral and uninvolved in the game itself, so would need to consist of people that don't play the game (ie. are not part of the players, can't vote on proposals, can't win etc.). With this system it might be possible to settle disagreements quicker, for example instead of the current issue where voting on the judgment will take place on Thursday, and if one of the judges doesn't vote this drags on even longer. Instead, one might opt for system where the court has the power to "pause" the game for a certain amount of time until the issue can be settled, and usually the game may go on - if the court considers (and no one disagrees) that the continuation of the game has no direct impact on the thing being judged [this is not part of the proposal, just an example of possible advantages].
In case that would be something wanted by the players, the members of the court might possibly also give comments on proposals on possible unintended and/or unclear consequences, and help fixing these things (restricted to the "technical" side, they should of course be neutral with regards to which game mechanisms are wanted etc.).
The court should probably be overruled by a unanimous vote by the players or something like that, but one disadvantage might be that it is possible that in some cases the ruling does not reflect the majority opinion of the players (that is however also not guaranteed in the current system). However, the majority can (if not contradicting immutable rules) change the rules as a reaction, and as the court is supposed to be neutral and uninvolved, hopefully this kind of thing does not advantage any particular player(s) in the long run. I also have the feeling that many inclarities are not of the form where it is even really clear who obtains an advantage or the advantage is quite minimal (for example in the current issue), so perhaps both sides could live with the material outcome of a ruling in either way (though they may disagree with the legal reasoning behind one of the two outcomes). For example in the current case, I have a definite opinion on the legal reasoning and what the outcome should be based on that (I am of course *not* neutral in this) but I actually could live quite well with the material outcome of a ruling saying my king is still on (0,0).

Whether or not people like this kind of idea depends a lot on which parts of the game they like best, so it is possible that people completely dislike this proposal. On that other hand, I could imagine that most other players might prefer the second part of the game over the first one. I personally also like the second one, but I also like the first one a lot, and I actually would like the opportunity to try to interpret the rules from a more neutral, non-involved standpoint. So I would be up for stopping to be a player and instead becoming a member of the court, if the (other) players wanted that. One comment back when we first briefly talked about this was that it might be boring for the judge, but I think we already had so much things in the rules that were unclear that that will not be a big problem, in particular if the court is also allowed to make comments on "loopholes" and technical problems with (proposed) rules on their own initiative, and weigh in on questions such as "whose turn is it" without there being a judgement invoked.
While I would be willing to act as a member of such a court, this is not an integral part of this idea; I mainly mentioned it because otherwise a comment might be "no one will want to be a member of such a court".

In case people like the general idea I would perhaps make a bit more specific/precise proposal based on feedback.

**Comment by Snark1994** [comment]
I enjoy both parts of the game, but the spirit of the game I would like to be playing is very much focussed on the new-ideas-and-doing-them rather than the legal-technicalities part of it. I am happy for you to become the "court" (this also allows mechanics which would need otherwise to be automated or left to trust, e.g. randomness), provided you are happy doing so too!
